src/iso19111/factory.cpp

```diff
@@ -7228,24 +7259,36 @@ AuthorityFactory::createFromCRSCodesWithIntermediates(
         const auto &auth_name2 = row[5];
         const auto &code2 = row[6];
         // const auto &accuracy2 = row[7];
-        auto op1 = d->createFactory(auth_name1)
-                       ->createCoordinateOperation(
-                           code1, true, usePROJAlternativeGridNames, table1);
-        if (useIrrelevantPivot(op1, sourceCRSAuthName, sourceCRSCode,
-                               targetCRSAuthName, targetCRSCode)) {
-            continue;
-        }
-        auto op2 = d->createFactory(auth_name2)
-                       ->createCoordinateOperation(
-                           code2, true, usePROJAlternativeGridNames, table2);
-        if (useIrrelevantPivot(op2, sourceCRSAuthName, sourceCRSCode,
-                               targetCRSAuthName, targetCRSCode)) {
-            continue;
-        }
+        try {
+            auto op1 =
+                d->createFactory(auth_name1)
+                    ->createCoordinateOperation(
+                        code1, true, usePROJAlternativeGridNames, table1);
+            if (useIrrelevantPivot(op1, sourceCRSAuthName, sourceCRSCode,
+                                   targetCRSAuthName, targetCRSCode)) {
+                continue;
+            }
+            auto op2 =
+                d->createFactory(auth_name2)
+                    ->createCoordinateOperation(
+                        code2, true, usePROJAlternativeGridNames, table2);
+            if (useIrrelevantPivot(op2, sourceCRSAuthName, sourceCRSCode,
+                                   targetCRSAuthName, targetCRSCode)) {
+                continue;
+            }
 
-        listTmp.emplace_back(
-            operation::ConcatenatedOperation::createComputeMetadata(
-                {op1, op2->inverse()}, false));
+            listTmp.emplace_back(
+                operation::ConcatenatedOperation::createComputeMetadata(
+                    {op1, op2->inverse()}, false));
+        } catch (const std::exception &e) {
+            // Mostly for debugging purposes when using an inconsistent
+            // database
+            if (getenv("PROJ_IGNORE_INSTANCIATION_ERRORS")) {
```

Comment:
I suggest to use the English spelling "instantiation" rather than the French(?) "instaciation"

Reply:
thanks. fixed